test/testUtils.ts

```diff
@@ -49,6 +50,15 @@ export function registerLocMocks(): void {
     });
 }
 
+/**
+ * Kills all running instances of chrome on the test host
+ */
+export function killAllChrome() {
+    const killCmd = (process.platform === 'win32') ? 'taskkill /F /IM chrome.exe /T' : 'killall chrome';
```

Comment:
Will this kill all the chrome.exe instances on my desktop every time I run the tests?

Reply:
Yes. Unless there's an easy way to get the PID of the instance that the debug adapter launched so we can just close that specific instance, I don't see another option other than just force killing all chrome instances here. Let me know if you know of a better way.

Reply:
I tested on my PC and "wmic process where ExecutablePath='path goes here\\chrome-win\\chrome.exe' delete" can kill only the chrome.exe instances at a specific path